### PR TITLE
Support running from stand-alone docker

### PR DESCRIPTION
**Pull request by ScottRoach**
## Summary
Support running the auto-discovery portion of this from a stand-alone docker container without relying on the add-ons. README is updated to explain how this can be run directly in docker as well. This addresses #76 

## Alternatives Considered
Starting a new repo that does the same thing as this repo, just without reliance on add-ons.

## Testing Steps
Running the following directly in Docker should allow the project to work similarly to the add-on.

```
docker run -e MQTT_HOST=mqtt.host.com -e MQTT_USERNAME=username -e MQTT_PASSWORD=password ghcr.io/pbkhrv/rtl_433-hass-addons-rtl_433_mqtt_autodiscovery-amd64
```

The add-on should continue to work as it currently does when it runs in add-on mode.

### CODE REVIEW

File: rtl_433_mqtt_autodiscovery/README.md [hunk 1]
```diff
@@ -103,3 +105,33 @@ The following options apply to all broker configurations:
 * `discovery_prefix`: MQTT topic prefix where Home Assistant is [looking for discovery information](https://www.home-assistant.io/docs/mqtt/discovery/#discovery_prefix). Default is "homeassistant".
 * `discovery_interval`: how often to publish discovery information, in seconds. Default is 600.
 * `force_update`: Use `force_update` flag in discovery configuration. See details [here](https://www.home-assistant.io/integrations/sensor.mqtt/#force_update).
+
+## Stand-alone Docker
+
+Run the following in Docker:
+
+```
+docker run -e MQTT_HOST=mqtt.host.com -e MQTT_USERNAME=username -e MQTT_PASSWORD=password ghcr.io/pbkhrv/rtl_433-hass-addons-rtl_433_mqtt_autodiscovery-amd64
```
Comment (deviantintegral) [inline rtl_433_mqtt_autodiscovery/README.md:114]: Let's use `mqtt.example.com` here.

**Review by deviantintegral — COMMENTED** [review]
One very minor suggestion, otherwise static review looks good! I hope to give this a try in a few days, but if someone else tests it ahead of me that's fine too.

File: rtl_433_mqtt_autodiscovery/README.md (hunk 1 above)
Comment (ScottRoach) [reply rtl_433_mqtt_autodiscovery/README.md]: This is updated!

**Review by deviantintegral — APPROVED** [review]
Works well for me both inside and outside of Home Assistant. I left a few more minor documentation suggestions. Otherwise, if we can pull `next` into your branch (looks like I don't have push rights) this should be good to merge. Thank you!

File: rtl_433_mqtt_autodiscovery/README.md [hunk 2]
```diff
@@ -103,3 +105,33 @@ The following options apply to all broker configurations:
 * `discovery_prefix`: MQTT topic prefix where Home Assistant is [looking for discovery information](https://www.home-assistant.io/docs/mqtt/discovery/#discovery_prefix). Default is "homeassistant".
 * `discovery_interval`: how often to publish discovery information, in seconds. Default is 600.
 * `force_update`: Use `force_update` flag in discovery configuration. See details [here](https://www.home-assistant.io/integrations/sensor.mqtt/#force_update).
+
+## Stand-alone Docker
+
+Run the following in Docker:
```
[the remark this answers is not in the document]
Comment (ScottRoach) [reply rtl_433_mqtt_autodiscovery/README.md]: Updated!

File: rtl_433_mqtt_autodiscovery/README.md [hunk 3]
```diff
@@ -103,3 +105,33 @@ The following options apply to all broker configurations:
 * `discovery_prefix`: MQTT topic prefix where Home Assistant is [looking for discovery information](https://www.home-assistant.io/docs/mqtt/discovery/#discovery_prefix). Default is "homeassistant".
 * `discovery_interval`: how often to publish discovery information, in seconds. Default is 600.
 * `force_update`: Use `force_update` flag in discovery configuration. See details [here](https://www.home-assistant.io/integrations/sensor.mqtt/#force_update).
+
+## Stand-alone Docker
```
[the remark this answers is not in the document]
Comment (ScottRoach) [reply rtl_433_mqtt_autodiscovery/README.md]: Updated!

**Comment by ScottRoach** [comment]
Related to this issue as well as [pbkhrv/rtl_433-hass-addons#82](https://github.com/pbkhrv/rtl_433-hass-addons/pull/82), I had a PR merged ([merbanan/rtl_433#1988](https://github.com/merbanan/rtl_433/pull/1988)) in the upstream (which #82 includes now) and think it would be a good use-case for this project. 

Are you good with me altering this PR to add support for a new field that would specify which device id's to watch? This would allow us to run this container constantly instead of only when we want to discover new items. The work would be based on both these PR's so I am open to other ways to try and implement this (if you are even interested in supporting it).

Cheers!

**Comment by deviantintegral** [comment]
@ScottRoach I like that! Let's do it as a followup pull request? I just merged both PRs to `next`.

**Pull request merged**